sites/partners/pages/index.tsx

```diff
@@ -129,12 +136,11 @@ export default function ListingsList() {
             <div className="flex justify-between">
               <div className="w-56"></div>
               <div className="flex-row">
-                {/* TODO, put behind a flag
                 <LocalizedLink href={`/listings/add`}>
                   <Button className="mx-1" onClick={() => false}>
```

Comment:
Is this onClick doing nothing because the parent tag is a link?

Reply:
Yes, but looking into it more, LocalizedLink is a legacy type that should probably be avoided. Updated to remove the parent and use onClick to redirect.